plugins/kubernetes-backend/src/kubernetes-auth-translator/AwsIamKubernetesAuthTranslator.ts

```diff
@@ -132,7 +132,7 @@ export class AwsIamKubernetesAuthTranslator
     );
 
     clusterDetailsWithAuthToken.serviceAccountToken = await this.getBearerToken(
-      clusterDetails.name,
```

Comment:
Maintains backwards compatibility. If `clusterName` isn't supplied default to the original `name` setting.